app/assets/stylesheets/config/_import.scss

```diff
@@ -0,0 +1,225 @@
+////////////////////////////////////////////////////////////////////////////////////
+// Colors. /////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////
+//
+// List of colors to be used to generate utility-frist classes,
+// using generate-classes() function. See files from /base folder
+// for reference.
+//
+// These colors are defined using some colors CSS variables defined
+// in config/_colors.scss. We only utilize some of these variables
+// for utility-classes because it doesn't make much sense to create
+// utility-class for ever single variable in config/_colors.scss.
+
+$colors: (
```

Comment:
This is a list of our "foundamental" colors so we can use them to generate utility-first classes.